docs/claimantmodel/README.md

```diff
@@ -0,0 +1,113 @@
+# Sigstore Claimant Model
+
+Sigstore's claimant model includes a set of claims produced by Rekor and Fulcio logs. See [Claimant Model](https://github.com/google/trillian/blob/master/docs/claimantmodel/CoreModel.md) for more information and terminology.
+
+Sequence diagrams generated with [Claimant Model Render Tool](https://github.com/google/trillian/tree/master/docs/claimantmodel/experimental/cmd/render). This tool is a work in progress, so there may be some errors in the generated models.
+
+Each claim is in `model.md` in each folder. `full.md` contains the claim made in both the domains of Rekor or Fulcio and the log. The sequence diagram shows the interactions between all Actors. 
+
+All claims are also provided in this readme.
+
+## Fulcio: Identity-based signing
+
+<dl>
+<dt>Claim<sup>Fulcio</sup></dt>
+<dd><i>${OIDCIDOwner} authorizes ${CA} to bind ${PubKey} to ${OIDCIdentity}</i></dd>
+<dt>Statement<sup>Fulcio</sup></dt>
+<dd>X.509 certificate containing ${PubKey} and ${OIDCIdentity}, signed by ${CA}</dd>
+<dt>Claimant<sup>Fulcio</sup></dt>
+<dd>${CA}</dd>
+<dt>Believer<sup>Fulcio</sup></dt>
+<dd>Software Installer</dd>
```

Comment:
Is Rekor also a Believer here?

Reply:
Agreed, there is no trusted action that Rekor takes based on a certificate. As long as you present a valid certificate whose public key can verify a signature, it will admit the entry into the log.